docs/architecture/how/gas.md

```diff
@@ -33,8 +33,12 @@ in more details.
 
 A signer pays all the gas required for a transaction upfront. However, there is
 no explicit act of buying gas. Instead, the fee is subtracted directly in NEAR
-tokens from the balance of the signer's account. If we ignore all the details
-explained further down, the fee is calculated as `gas amount` * `gas price`.
+tokens from the balance of the signer's account. The fee is calculated as `gas
+amount` * `gas price`.
```

Comment:
Can we add something like:
Gas amount for actions included in a `SignedTransaction` are all fixed, except for function calls where the user needs to specify the amount.

possibly with a link to here: https://docs.near.org/protocol/gas#cost-for-common-actions